crates/web5/src/lib.rs

```diff
@@ -4,3 +4,30 @@ pub mod dids;
 
 #[cfg(test)]
 mod test_helpers;
+
+lazy_static::lazy_static! {
```

Comment:
could we use the `LazyLock` or `LazyCell` here and then remove the addition of `lazy_static` in the `Cargo.toml`

Reply:
Upd8d